backends/platform/sdl/win32/win32.cpp

```diff
@@ -192,7 +190,7 @@ Common::String OSystem_Win32::getScreenshotsPath() {
 	char picturesPath[MAXPATHLEN];
 
 	// Use the My Pictures folder.
-	if (SHGetFolderPath(NULL, CSIDL_MYPICTURES, NULL, SHGFP_TYPE_CURRENT, picturesPath) != S_OK) {
+	if (!SHGetSpecialFolderPath(NULL, picturesPath, CSIDL_MYPICTURES, FALSE)) {
```

Comment:
This function is marked as unsupported in MSDN

Reply:
Well I doubt they support anything 9x era, but from my testing it still works as recently as Windows 10. If you think it's unreliable I can dynamically link to the function instead (a version check isn't enough since Win32 calls are linked at compile-time).

Reply:
Indeed, it still works, and Microsoft is known to try and keep backwards compatibility with wrapper functions.

Reply:
And you are correct about the compile-time checks. How would you link dynamically to it?

Reply:
You have to use `GetProcAddress` to get a function pointer instead, you can see an example in the Win32TaskbarManager: https://github.com/scummvm/scummvm/blob/master/backends/taskbar/win32/win32-taskbar.cpp#L381

Reply:
I believe that this is a good way to tackle this: check if SHGetFolderPath() exists and use it, otherwise fall back to SHGetSpecialFolderPath() for Windows 9x

Reply:
Just wanted to chime in and say that detecting and calling the new function with GetProcAddress() and linking to and falling back on the old one is the right way to handle this. Also, fanatics who care about keeping things compiling on Win98 are very much my people. =)

Reply:
@bluegr Replaced with the appropriate check.